Dockerfile

```diff
@@ -1,18 +1,56 @@
-FROM python:3.6
+### Build python virtualenv
+FROM python:3.6-stretch as build-python
 ENV PYTHONUNBUFFERED 1
 
 RUN \
   apt-get -y update && \
-  apt-get install -y gettext && \
-  apt-get clean
+  apt-get install -y gettext python3-virtualenv virtualenv && \
+  apt-get clean && \
+  rm -rf /var/lib/apt/lists/*
```

Comment:
You could maybe add a comment saying `empty the APT cache`?

Reply:
Ok